### PR TITLE
Module 5 exercise

### PR DESCRIPTION
**Pull request by lukestern**
I had to make a correction on module 3 and then rebased the module 4 & 5 branches to the new module 3 commit. It looks like this has not changed the module 3 & 4 pull requests but I have ended up with some duplication on this pull request. 

Commits on April 13th and 14th are all module 4.
Commits on April 26th are all module 5.
Commits on April 27th are all these three modules combined.

I hope that makes sense and simplifies looking over this pull request .

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -3,6 +3,7 @@ FLASK_APP=todo_app/app
...
 
 # Trello Details
-TRELLO_BOARD_ID = board_id
-TRELLO_KEY = key
-TRELLO_TOKEN = token
\ No newline at end of file
+TRELLO_BOARD_ID=board_id
+TRELLO_KEY=key
+TRELLO_TOKEN=token
+ID_ORGANIZATION=workspace_name
```
Comment (BenRamchandani) [inline .env.template:9]: Would be good to add a note to your README

File: docker-compose-test.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+  todo-app-tests:
+    build: 
+        context: .
+        dockerfile: Dockerfile
+        target: test
+    image: todo-app:test
+    ports: 
+        - "5001:5001"
```
Comment (BenRamchandani) [inline docker-compose-test.yml]: Do you need to expose these ports?

**Review by BenRamchandani — APPROVED** [review]
Couple of minor comments, but looks good :+1:

File: .env.template (hunk 1 above)
Comment (lukestern) [reply .env.template:9]: README updated

File: docker-compose-test.yml (hunk 2 above)
Comment (lukestern) [reply docker-compose-test.yml]: Removed

**Pull request merged**